components/ChipTextField/examples/MDCChipTextField.h

```diff
@@ -12,8 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#import "MDCChipView.h"
 #import "MDCTextField.h"
 
+@class MDCChipTextField;
+
+@protocol MDCChipTextFieldDelegate <NSObject>
+
+@optional
```

Comment:
I'm not sure how much I like a protocol that all its methods are optional. One way to avoid it is to use multiple protocols, or perhaps an aggregator of sub protocols. Let's rethink this.

Reply:
Followed up offline. Since these methods are optional, we need to keep make sure it is explicit.